cloud-aws-cloudformation/src/main/resources/definitions/aws-zone-coordinates.json

```diff
@@ -114,7 +114,14 @@
     },
     {
       "name": "us-gov-west-1",
-      "displayName": "GovCloud (US)",
+      "displayName": "AWS GovCloud (US-West)",
```

Comment:
Couldn't this new gov cloud realted zones be stored in the gov cloud related module and variant?

Reply:
I think this is also in common module so we should just remove the entirely file from here.